tests/TestCase/ORM/Query/SelectQueryTest.php

```diff
@@ -999,6 +999,27 @@ public function testApplyOptions(): void
         $this->assertEquals($expected, $query->getContain());
     }
 
+    public function testApplyOptionsSelectWhere(): void
+    {
+        $options = [
+            'select' => ['field_a', 'field_b'],
+            'where' => ['field_a' => 1, 'field_b' => 'something'],
+        ];
+        $query = new SelectQuery($this->table);
+        $query->applyOptions($options);
```

Comment:
I thought we were trying to make finder options go away? A long time ago they were intended as a way to help upgrade from 2.x `find()` options. I would rather we find a way to remove the existing options over adding more.

Reply:
Yes we do want them to go away, though we haven't been able to come up with a solution yet 🙂.